tutorials/FITS-cubes/FITS-cubes.ipynb

```diff
@@ -0,0 +1,548 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3D Data Cubes\n",
+    "In the tutorial we will visualyse 2D and 3D data sets in different coordinates (galactic and ecuatorial). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Library imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from astropy.utils.data import download_file\n",
```

Comment:
I'm not sure what's been done in other tutorials, but while in a script it's advisable to import everything at the top, in a tutorial it may make more sense to import things where they are first used? 

Also clustering imports are usually advisable, first standard library, then 3rd party, maybe here clusters by packages?